python/cuml/cuml/manifold/spectral_embedding.pyx

```diff
@@ -244,6 +366,79 @@ class SpectralEmbedding(Base,
             "n_neighbors"
         ]
 
+    @classmethod
+    def _params_from_cpu(cls, model):
+        """Get parameters to use to instantiate a GPU model from a CPU model.
+
+        Parameters
+        ----------
+        model : sklearn.manifold.SpectralEmbedding
+            The CPU model to get parameters from
+
+        Returns
+        -------
+        dict
+            Parameters to pass to the GPU model constructor
+        """
+        affinity = getattr(model, 'affinity', 'nearest_neighbors')
+        if affinity != 'nearest_neighbors':
+            raise UnsupportedOnGPU(
+                f"`affinity={affinity!r}` is not supported. "
+                "Only 'nearest_neighbors' affinity is currently supported."
+            )
+
+        eigen_solver = getattr(model, 'eigen_solver', None)
+        if eigen_solver is not None and eigen_solver not in [None, 'arpack', 'lobpcg', 'amg']:
+            raise UnsupportedOnGPU(
+                f"`eigen_solver={eigen_solver!r}` is not supported."
+            )
```

Comment:
Since the different solvers shouldn't result in different results (just different ways of achieving them) we tend to ignore parameters like this entirely rather than falling back.

```suggestion
```